sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/PipelinedDocumentQueryParams.java

```diff
@@ -39,7 +42,8 @@ public PipelinedDocumentQueryParams(
         List<PartitionKeyRange> partitionKeyRanges,
         QueryInfo queryInfo,
         CosmosQueryRequestOptions cosmosQueryRequestOptions,
-        UUID correlatedActivityId) {
+        UUID correlatedActivityId,
+        List<FeedRangeEpkImpl> feedRanges) {
```

Comment:
seems we are still passing `partitionKeyRanges` along `feedRanges`.
are both needed?

Reply:
Nope. Cleaned up the partitionKeyRanges